### PR TITLE
Fixes for TC's phpcs

### PR DESCRIPTION
**Pull request by obenland**
### Changes proposed in this Pull Request:
Fixes phpcs errors when trying to merge NHA to wp.com

See D36505-code

### CODE REVIEW

File: class-newspack-blocks-api.php [hunk 1]
```diff
@@ -60,11 +60,14 @@ public static function register_rest_fields() {
...
-	 * @param Array $object  The object info.
+	 * @param array $object The object info.
+	 * @return array Featured image.
 	 */
 	public static function newspack_blocks_get_image_src( $object ) {
+		$featured_image_set = [];
+
 		if ( 0 === $object['featured_media'] ) {
-			return;
+			return $featured_image_set;
```
Comment (jeryj) [inline class-newspack-blocks-api.php]: Will changing this from returning `null` to an empty array have any unintended consequences? I searched the repo for  `newspack_blocks_get_image_src` but only found it referenced here.

Comment (jeffersonrabb) [reply class-newspack-blocks-api.php]: Good  catch, I think you may be right. This value is handled here  https://github.com/Automattic/newspack-blocks/blob/master/src/blocks/homepage-articles/edit.js#L102-L131, and it looks like an empty array will in fact bork the logic. I'll make the change.

Comment (jeryj) [reply class-newspack-blocks-api.php]: Thanks for referencing that code. I wasn't sure where it ended up being used.

Comment (jeffersonrabb) [reply class-newspack-blocks-api.php]: Addressed in https://github.com/Automattic/newspack-blocks/pull/271/commits/c4f77090636fab49f1c7f4458c0dde64c0d22be9. Could you verify this causes no new TC issues?

Comment (obenland) [reply class-newspack-blocks-api.php]: It'll be fine, the original complained was that the variable was undefined, which is still fixed. Thanks for catching that, it looked to me like it was just a filter callback.

**Pull request merged**